internal/pkg/manifest/env.go

```diff
@@ -270,7 +271,19 @@ func (cfg *EnvironmentCDNConfig) UnmarshalYAML(value *yaml.Node) error {
 	return nil
 }
 
-// IsEmpty returns true if vpc is not configured.
+// CDNStaticConfig represents the static config for CDN.
+type CDNStaticConfig struct {
+	Location string  `yaml:"location,omitempty"`
```

Comment:
Does this work nicely with environment overrides `mergo`?

Reply:
This is an env manifest field. How do we use env overrides 🤔

Reply:
:homer-disappearing